api/internal/features/domain/controller/get_domains.go

```diff
@@ -15,10 +16,16 @@ import (
 )
 
 func (c *DomainsController) GetDomains(f fuego.ContextNoBody) (*shared_types.Response, error) {
-	organization_id := f.QueryParam("id")
-
 	w, r := f.Response(), f.Request()
 
+	organization_id := utils.GetOrganizationID(r)
```

Comment:
this change will break the client(frontend) which is using it, so make sure to apply the changes wherever applicable

Reply:
Umm, I don't see any way this might pose as an issue in the frontend. we previously were accepting the query string, now we are using it directly from user-context from the token, which ideally makes more sense, na? Otherwise any user with API access could retrieve domain data for any organization in the system for that hosted instance.


In case you meant to remove passing id as query strings in the frontend, can do that.

Lmk otherwise if im missing some thing

Reply:
> In case you meant to remove passing id as query strings in the frontend, can do that.

yes this actually